test/quantization/test_quantize_fx.py

```diff
@@ -266,7 +266,35 @@ def forward(self, x):
         m = convert_static_fx(m)
         m(dict_input)
 
+    @skipIfNoFBGEMM
+    def test_qconfig_none(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.conv1 = nn.Conv2d(1, 1, 1)
+                self.conv2 = nn.Conv2d(1, 1, 1)
+
+            def forward(self, x):
+                x = self.conv1(x)
+                x = self.conv2(x)
+                return x
 
+        m = M().eval()
+        m = symbolic_trace(m)
+        qconfig_dict = {'': default_qconfig, 'conv2': None}
```

Comment:
just curious, what should we say the behavior is in a case where there are nodes (A -> B -> C) in the source graph, a rule which matches (A -> B -> C), and one of nodes {A, B, C} has qconfig=None?  Should we make the matching rules respect that all nodes must have qconfig != None for a pattern to be valid?

Reply:
yeah i have a similar comment in the api doc for fusion as well, on whether it should respect qconfig or not.
i think the rules need to make sure the qconfig for the matched nodes are the same before marking it as a match, similar for fusion. although this is not a high pri since it sounds like a rare use case.

Reply:
makes sense, as long as it's easy to change later if the use case pops up, without breaking BC, etc